internal/xds/server.go

```diff
@@ -14,12 +14,17 @@
 package xds
 
 import (
+	"context"
+
+	api "github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
How about keeping this import as `api` and removing the other duplicates?

Reply:
Was just copy pasting from envoy example code, works to shorten the imports.